performance-tests/bench/example/config/worker/bollea_aggregator.json

```diff
@@ -142,10 +142,10 @@
       "writers": [ "datawriter_01" ],
       "params": [
         { "name": "data_buffer_bytes",
-          "value": { "_d": "PVK_ULL", "ull_prop": 2372 }
+          "value": { "$discriminator": "PVK_ULL", "ull_prop": 2372 }
```

Comment:
We will also need to update the JSON configs in nightly for the performance dashboard. Ideally, we'd check in those changes just before merging this to master so the dashboard run pulled the right format JSON.